dali/pipeline/workspace/sample_workspace.h

```diff
@@ -72,6 +72,12 @@ class DLL_PUBLIC SampleWorkspace : public WorkspaceBase<SampleInputType, SampleO
   template <typename Backend>
   DLL_PUBLIC Tensor<Backend>& Output(int idx);
 
+  int GetInputBatchSize(int) const {
+    DALI_FAIL(
+        "Impossible function: Sample workspace is not aware, that there exists such thing as an "
```

Comment:
```suggestion
        "Not implemented: Sample workspace is not aware, that there exists such thing as an "
``?

Reply:
Can you make it a compile time assert if anyone uses that function?

Reply:
I still don't like the idea of having GetInputBatchSize at all, but I won't sweat it.
Regarding this function - why is it here? If it's wrong to call it - then it should not be called at all. In that case, why is it here at all?